hypothesis-python/tests/array_api/README.md

```diff
@@ -30,3 +33,15 @@ or use the import path (**2.**),
 The former method is more ergonomic, but as entry points are optional for
 adopting the Array API, you will need to use the latter method for libraries
 that opt-out.
+
+
+## Running against different API versions
+
+You can specify the `api_version` to use when testing array modules via the 
+`HYPOTHESIS_TEST_ARRAY_API_VERSION` environment variable. There is one
+recognized option:
+
+* `"default"`: infers the latest API version for each array module.
+
+Otherwise the test suite will use the variable as the `api_version` argument for
+`make_strategies_namespace()`.
```

Comment:
Short note that we intend to add support for running against all versions supported by a library in future?  Don't need to actually *do* that now, but it'd be good to clarify that we do want that.